tests/integration/src/lib.rs

```diff
@@ -25,3 +25,6 @@ mod setup;
 
 #[cfg(any(feature = "shibuya", feature = "shiden", feature = "astar"))]
 mod proxy;
+
+#[cfg(feature = "shibuya")]
+mod xvm;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Consider renaming the modules `proxy` and `xvm` to have more meaningful names that reflect their purpose or functionality. This can improve code readability and maintainability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->